stable/pipelines/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # JFrog Pipelines Chart Changelog
 All changes to this chart to be documented in this file.
 
+## [1.3.9] - July 31, 2020
+* Added support for customVolumes, configMaps and customInitcontainers for Vault
```

Comment:
What about customVolumeMounts?

Reply:
@danielezer These are already done for `pipelines` may not needed for `vault`

Reply:
What would you use custom volumes for if you don’t have custom volume mounts

Reply:
Added